app/actions/index.js

```diff
@@ -276,3 +280,46 @@ export const listRemoteSurveys = () => (dispatch, getState) => {
       );
   });
 };
+
+export const initializeObservation = tags => dispatch =>
+  dispatch({
+    type: types.INITIALIZE_OBSERVATION,
+    tags
+  });
+
+export const updateObservation = tags => dispatch =>
+  dispatch({
+    type: types.UPDATE_OBSERVATION,
+    tags
+  });
+
+export const saveObservation = () => (dispatch, getState) => {
+  const { observation } = selectActiveObservation(getState());
+
+  dispatch({
+    type: types.SAVING_OBSERVATION,
+    observation
+  });
+
+  // TODO this is wrong
+  // observation currently looks like (still needs a point / associated feature):
+  // {
+  //   tags: {
+  //     highway: "residential",
+  //     name: "My Street",
+  //     surface: "paved"
+  //   }
+  // }
+  return osm.createObservation(observation, error => {
```

Comment:
https://github.com/osmlab/field-data-collection/blob/master/app/lib/osm-p2p.js#L64 confuses me then, since the signature is `nodeId, geojson, opts, cb`.

Should I be using a different function?